posix-configs/eagle/200qx/mainapp.config

```diff
@@ -1,9 +0,0 @@
-uorb start
```

Comment:
@jgoppert it seems like these files got removed instead of moved.

I thought *.config was removed from the gitignore, strange.